src/masonite/orm/blueprint/Blueprint.py

```diff
@@ -64,13 +64,30 @@ def __init__(self, column, constraint_type):
         self.constraint_type = constraint_type
 
 
+class ForeignKey:
+    def __init__(self, column, current_table, foreign_table=None, foreign_column=None):
+        self.column_name = column
+        self.current_table = current_table
+        self.foreign_table = foreign_table
+        self.foreign_column = foreign_column
+        self.index_name = ""
+
+    def references(self, foreign_column):
+        self.foreign_column = foreign_column
+
+    def on(self, foreign_table):
+        self.foreign_table = foreign_table
+        self.index_name = f"{self.current_table}_{self.column_name}_foreign"
+
```

Comment:
I think we should add the `on_delete` and `on_update` methods

Reply:
Yes also right